liblangutil/Token.h

```diff
@@ -326,6 +326,8 @@ namespace TokenTraits
 			tok == Token::TrueLiteral || tok == Token::FalseLiteral;
 	}
 
+	bool isYulKeyword(std::string const& _literal);
```

Comment:
Using `isYulKeyword` here, hence the other helper is called `isYulToken`. Open to other naming suggestions.